node_modules/.yarn-integrity

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
What is this file? In general we don't want to add things inside node_modules, as they may be build specific

Reply:
This somehow got created when I ran yarn install. Not sure how to exclude it from the pull request.

Reply:
It happens, especially if you use `git commit -a`.

Since it's in node_modules it's fine to blow away and rebuild. Try:
`git rm node_modules/.yarn-integrity`

and then commit your change. BTW [this website](https://git-scm.com/docs/git-rm) is really helpful for understanding git.